en/tips/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+
+### 2. Try to fix the database schema
+
+Go to `http://[your-mautic-url]/s/update/schema`. It will update your database schema if there are some known updates to be applied.
+
+## Nope, it didn't help
+
+Alright. Let's figure out what the issue really is.
+
+### Check the logs
+
+#### Mautic logs
+
+Please, do not create a new issue/post/comment anywhere where you'll post that the error message is:
+
+> The site is currently offline due to encountering an error. If the problem persists, please contact the system administrator
+
+or
+
+> Uh oh! I think I broke it.
+
+Those are generic error messages which provide absolutely no valuable information about the cause of the error. It is on purpose. The detailed error message isn't displayed in the production environment because if it would, it could tell possible cracker a valuable information about your server which he/she could use to attack it. That's why this generic error message is shown instead and the real error message is logged into the logs and that's why posting it doesn't make any sense.
+
+There are different logs in your system which could tell us more. Let's start form the Mautic log.
+
+If your Mautic administration work, go to the Admin menu (click the __cog icon__ in the top right side corner), then __System Info__, then __Log_. You will see the error messages recorded by Mautic today.
+
+If your Mautic administration doesn't work, open the logs in via the file system. Go to `[mautic_root]/app/logs` folder. You should see one file for every day called `mautic_prod-YYYY-mm-dd.php`. Open the latest one.
+
+#### Server logs
+
+I don't have experiences with all operation systems and servers. I can tell you that Apache2 saves the server logs at Ubuntu in `/var/log/apache2/error.log`. If you don't know where the server logs are stored, contact your sysadmin or try to google it.
+
+#### MySql logs
+
+In case your Mautic says it is not able to connect to MySql for some reason, you can check what information is stored in the MySql log. Again, I can only advice where the log is in Ubuntu: `/var/log/mysql/error.log`. But to be honest, the database connection error is in 90% caused by wrong credentials.
+
+#### The log file is too big
+
+In case the log is too big so a normal editor cannot open it and you have the CLI access, try to read only the few last rows of it with command `tail [mautic_root]/app/logs/mautic_prod-YYYY-mm-dd.php`. You have to add the current date in the instead of the `YYYY-mm-dd` part. If you want to see more rows than the default is, use the `-n 40` attribute at the end of the command to see last 40 rows.
+
+#### But I don't understand the error message
+
+Take a deep breath and try to actually read the error messages. Some error messages actually have the solution written in them. Look at this one:
```

Comment:
Maybe instead of "Look at this one:" then "Or this one:", do something like "Let's look at some examples:" and list them out like:

**Error:** 
```
Fatal: Automatically populating $HTTP_RAW_POST_DATA is deprecated and will be removed in a future version. To avoid this warning set 'always_populate_raw_post_data' to '-1' in php.ini and use the php://input stream instead. - in file Unknown - at line 0 [] []
```
**Solution:** Set 'always_populate_raw_post_data' to '-1' in php.ini


**Errors**: 
```
PHP Fatal error:  Allowed memory size of 67108864 bytes exhausted (tried to allocate 10924085 bytes) in ...
```
```
Maximum execution time of 120 seconds exceeded - in file ...
```
**Solution:** This means Mautic needs more memory / time than your server limit is. Again, contact your sysadmin and consult your possibilities.